postgres-init/README.md

```diff
@@ -0,0 +1,31 @@
+# Postgres Upgrade
+## Overview
+* Used to upgrade the schemas and required data from one MOSIP version to another.
+* It can upgrade one version at a time.
+* In case the need is to upgrade multiple versions, the same needs to be updated sequentially in upgrade.csv.
```

Comment:
In case, you want to upgrade multiple versions, the same needs to be updated sequentially in `upgrade.csv`.